src/scss/style.scss

```diff
@@ -1,14 +1,202 @@
+$primaryColor: rgb(233, 229, 229);
```

Comment:
Decide on one way to display colors and use that consistently.

Reply:
alright.